HW7/18.py

```diff
@@ -0,0 +1,13 @@
+# задача 34 вариант решения Вини-пух(па папа па)
+data = input().split()
+print(data)
+
+def funk(D):
+    x = [list(D[x]).count('а') for x in range(len(D))]
```

Comment:
Во-первых, зачем  `list` 
```suggestion
    x = [D[x].count('а') for x in range(len(D))]
```
Во-вторых, зачем  `range` 
```suggestion
    x = [x.count('а') for x in D]
```
В-третьих, а как же остальные гласные?